content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -104,7 +104,7 @@ To mitigate the impact of short history window, we introduced a concept of `book
           "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "12746"} }
         }
 
-`Bookmark` events can be requested by `allowWatchBookmarks=true` option in watch requests, but clients shouldn't assume bookmarks are returned at any specific interval, nor may they assume the server will send any `bookmark` event. As of 1.15 release, it is an Alpha feature.
+`Bookmark` events can be requested by `allowWatchBookmarks=true` option in watch requests, but clients shouldn't assume bookmarks are returned at any specific interval, nor may they assume the server will send any `bookmark` event. As of 1.16 release, it is a Beta feature.
```

Comment:
hello @wojtek-t. The updates seem fine. What do you think about using the feature-state shortcode, such as
{{< feature-state for_k8s_version="v1.16" state="beta" >}}, for the watch/bookmark feature? As it stands, the feature state and version are hard to locate in the paragraph.
Also, should this feature/state be listed in the feature gates table,
https://5d397c09ad55c5000cd80afd--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/

Reply:
@kbhawkey - thanks for suggestion. I applied the comment.

PTAL if the current state is what you expected.

Reply:
hello, @wojtek-t  
one nit: use back ticks around boolean flag, Default column, in feature gates table, for example:
`true`
https://5d3b57ae198fa600097ff5b0--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/